packages/runner/src/run.ts

```diff
@@ -376,7 +376,7 @@ export async function runSuite(suite: Suite, runner: VitestRunner): Promise<void
 
   suite.result = {
     state: 'run',
-    startTime: start,
+    startTime: Date.now(),
```

Comment:
Why?

Reply:
Hi! @sheremet-va 👋

Date.now() gives us milliseconds since 1970, but performance.now() doesn't. 
The name 'startTime' implies it might be used as a specific timestamp, like in `test/cli/test/reported-tasks.test.ts:106`.
```
const diagnostic = passedTest.diagnostic()!
expect(diagnostic).toBeDefined()
expect(diagnostic.heap).toBeGreaterThan(0)
expect(diagnostic.duration).toBeGreaterThan(0)
expect(date(new Date(diagnostic.startTime))).toBe(date(now))
expect(diagnostic.flaky).toBe(false)
expect(diagnostic.repeatCount).toBe(0)
expect(diagnostic.repeatCount).toBe(0)
```
If we switch to performance.now(), we'll probably need to update that test file. So, do I need to do this?

Reply:
Yes, I would expect it to use the new `now`

Reply:
Thanks for the review! I've made the changes you suggested.

Reply:
Hey @sheremet-va 👋,

I've noticed something important about our test execution, particularly for files like `test/reporters/tests/merge-reports.test.ts`. We're using `Tinypool` from `packages/vitest/src/node/pools/forks.ts` to spawn new processes for these tests.

The issue is with our use of `performance.now()`. While it works fine for timing within a single process, it becomes problematic across multiple processes. Each process has its own time reference for `performance.now()`, starting from when that process begins.

This causes issues when we try to use these `startTime` values for sorting in `packages/vitest/src/node/reporters/blob.ts`. The times from different processes aren't comparable.

```
  const files = blobs
    .flatMap(blob => blob.files)
    .sort((f1, f2) => {
      const time1 = f1.result?.startTime || 0
      const time2 = f2.result?.startTime || 0
      return time1 - time2
    })
```

Given this, I still think we should use `Date.now()` for `startTime`. It provides a consistent timestamp across all processes.

What are your thoughts on this? Do you see any potential drawbacks to switching to `Date.now()`?

Reply:
Ready for review! 🚀